.kitchen.yml

```diff
@@ -42,23 +40,33 @@ provisioner:
       base:
         '*':
           - systemd
-  state_top:
-    base:
-      '*':
-        - systemd
-        - systemd.networkd
-        - systemd.networkd.profiles
 
 verifier:
   name: serverspec
-  use_sudo: yes
-  sudo_path: true
 
 suites:
-  - name: default
+  - name: centos
     verifier:
       default_pattern: true
-      bundler_path: '/usr/local/bin'
-      rspec_path: '/usr/local/bin'
-      env_vars:
-        SUDO: true
+    includes:
+      - centos-7
+    provisioner:
+      state_top:
+        base:
+          '*':
+            - systemd
+            - systemd.networkd
+            - systemd.networkd.profiles
+  - name: ntp
```

Comment:
I'd keep it named as `default` and add an

```yaml
excludes:
  - centos-7
```
so it's run in all the other platforms but the one not supporting it?

Reply:
I started with a similar solution, but having to run two suites on Debian ("default" + "ntp") takes more time., that's why I did this.